### DISCUSSION TITLE
0.4.0 Spell checking

### DISCUSSION THREAD

**Discussion by defkev** [florisboard/florisboard#2594] [Q&A] [answered]
I just upgraded to 0.4.0 (from 0.3.16) on F-Droid and spell checking is gone?

The spell checker service (in system) is still there but the option to manage Hunspell dictionaries (Mozilla xpi) is missing, e.g. everything below https://github.com/florisboard/florisboard/wiki/System-spell-checker-service#how-do-i-add-dictionaries-for-spell-checking no longer works?

Or am i taking crazypills?!

**Comment by patrickgold** [comment, marked as answer]
Hey,

the spell checker thing with Hunspell is indeed gone, you are not on crazypills :) The reason for that was licensing issues (long story). But I forgot to update the wiki, so thanks for this reminder!

A new spell checker is coming alongside word predictions in the 0.5 dev cycle which uses the same base dict as the word predictions.
